src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,125 @@
+    - Parameterized repeated span description: Using parameterized descriptions of the repeated span in the fingerprint.
+    - Parameterized source span description: Including parameterized descriptions of the source span in the fingerprint.
+    - Parameterized parent span description + parent operation: Incorporating parameterized descriptions of the parent span, along with the parent operation, in the fingerprint.
+
+By utilizing this fingerprinting approach, we can effectively identify and classify N+1 database queries based on their unique characteristics and contextual information, ensuring effective grouping.
+
+# Actionability/Usefulness
+
+Once you have figured out the fingerprinting process and know how to group instances of the same problem with the same fix, the next step is to provide users with all the necessary information to fix the issue. The quality of an issue lies not only in its accurate detection but also in its actionability.
+
+N+1 DB Query detection relies heavily on span analysis to identify problematic spans and guide users toward a potential fix.
+
+Reporting circumstantial data is not useful! This is a crucial distinction between issues and alerts. Issues should pinpoint root problems and help resolve them.
+![Untitled](img/issuePlatformFingerprinting01.jpg)
+
+# Signal vs. Noise
+
+With every new issue type, we could be adding more and more noise to the Issues product. New issues fatigue is real for our customers. This is why it’s important to understand how a new detector you want to ship affects the overall noise of the Issue Stream.
+
+Accurate detection of truly valuable problems and correct grouping can transform "noise" into "signal." Every detector should be evaluated against this standard.
+
+# Bits’n’Bobs
+
+Performance issues affect many parts of the system, and it’s easy to overlook something. Here are a few places to look to make sure your issues are working well:
+
+- **UI edges**. Do your issues look correct in the issue stream? Check the severity level, title, description, and hover card content.
+- **Email notifications**. Does the UI match the issue details page for your new detector’s issues?
+- **Slack notifications**. Do the notifications for your issue make sense?
+- **Integrations**. Does converting your issue to a Jira/Linear/GitLab task work nicely?
+- **Searching**. Is is possible to search for your issue by type in the issue stream?
+- **Alpha and Beta badges**.  Did you add Alpha or Beta badges to the issue details page as appropriate?
+- **Public documentation**. Does your detector have a corresponding page on [docs.sentry.io](https://docs.sentry.io)?
+
+# Testing
+
+Whenever possible, conduct tests using real data. It's advised to use actual transaction event JSON files to perform unit testing for Performance Detectors.
+
+Another tool that we used for validating and testing detector changes is a CLI script: https://github.com/getsentry/sentry/pull/39727
```

Comment:
```suggestion
Another tool that we used for validating and testing detector changes is a CLI script:
    ```text
    # https://github.com/getsentry/sentry/pull/39727
    sentry performance detect /path/to/event.json
    ```
```